app/segments/drill/[id]/submission/input.js

```diff
@@ -262,7 +262,7 @@ function createOutputData(drillInfo, inputValues, attemptShots, uid, did) {
 
         case "sideLanding":
           shot.sideLanding = Number(inputValues[j].sideLanding);
-          sideLandingTotal += Number(inputValues[j].sideLanding);
+          sideLandingTotal += Math.abs(Number(inputValues[j].sideLanding));
```

Comment:
I think sideLandingTotal should not be absolute values, but sideLandingAverage should

Reply:
I figured you would want the total to be absolute values as well for the same reason. Want to know how far overall you are missing from straight rather than a left and right. Say you miss equal distance left and right, your total would be 0 and would seems as you were always hitting straight

Reply:
Shouldn't that be what the shotTendency chart's job? can we make it work for Line Test?

Reply:
I guess that makes sense. If you're negative, you are hitting left more, if it's positive, you miss right more. We talked about getting the shotTendency working for lineTest, so I can maybe figure that out